export_to_godot_tilemap.js

```diff
@@ -77,7 +77,13 @@ class GodotTilemapExporter {
             this.extResourceId = index + 1;
             this.tilesetsIndex.set(tileset.name, this.extResourceId);
             // noinspection JSUnresolvedVariable
-            let tilesetPath = tileset.asset.fileName.replace(this.projectRoot, "").replace('.tsx', '.tres');
+            let tilesetPath = tileset.asset.fileName
+            let slashIndex = tileset.asset.fileName.lastIndexOf("/");
+            if(slashIndex >= 0) {
+                let filePath = tileset.asset.fileName.substring(0, slashIndex);
```

Comment:
Since you introduced a variable, let's use it where possible:
```suggestion
            let slashIndex = tilesetPath.lastIndexOf("/");
            if(slashIndex >= 0) {
                let filePath = tilesetPath.substring(0, slashIndex);
```
Also, instead of looking for `/`, consider using [`FileInfo.path(tilesetPath)`](https://www.mapeditor.org/docs/scripting/interfaces/FileInfo.html#path). Though it's not clear to me if we need to handle the case where there isn't a `/`. This should actually only happen in case of an embedded tileset, which has an empty filename.